### DISCUSSION TITLE
Some errors about uploading Adafruit Feather nRF52840 using Arduino IDE 2.0.4

### DISCUSSION THREAD

**Discussion by HTseo** [adafruit/Adafruit_nRF52_Bootloader#296] [Q&A]
Hi.

I have some errors about uploading Adafruit Feather nRF52840 using Arduino IDE 2.0.4.

When I upload a code, one time is success. next fail.
then I change bootloader, after that, one time is success. next fail.
then...
sometimes success is twice, but repeatd fail and success the upload process.

this routine is repeated mostly.
What is the problem on my process?

**Comment by hathach** [comment]
which bootloader versin you are running, double reset the board and paste your INFO_UF2.TXT here
